src/main/scala/is/hail/keytable/KeyTable.scala

```diff
@@ -137,29 +113,9 @@ case class KeyTable(@transient hc: HailContext, @transient rdd: RDD[(Annotation,
     }
   }
 
-  def mapAnnotations[T](f: (Annotation) => T)(implicit tct: ClassTag[T]): RDD[T] =
-    KeyTable.toSingleRDD(rdd, nKeys, nValues).map(a => f(a))
-
-  def mapAnnotations[T](f: (Annotation, Annotation) => T)(implicit tct: ClassTag[T]): RDD[T] =
-    rdd.map { case (k, v) => f(k, v) }
+  def mapAnnotations[T](f: (Annotation) => T)(implicit tct: ClassTag[T]): RDD[T] = rdd.map(a => f(a))
 
-  def query(code: String): (Type, (Annotation, Annotation) => Any) = {
-    val ec = EvalContext(fields.map(f => (f.name, f.typ)): _*)
-    val nKeysLocal = nKeys
-    val nValuesLocal = nValues
-
-    val (t, f) = Parser.parseExpr(code, ec)
-
-    val f2: (Annotation, Annotation) => Any = {
-      case (k, v) =>
-        KeyTable.setEvalContext(ec, k, v, nKeysLocal, nValuesLocal)
-        f()
-    }
-
-    (t, f2)
-  }
-
-  def querySingle(code: String): (Type, Querier) = {
+  def rowQuerier(code: String): (Type, Querier) = {
```

Comment:
The analogous functions on VSM are called `queryVA`, `queryVariants`, `queryGlobal`, etc.  This should match.  Just `query` or `queryRow`?